.github/workflows/ruleset-checks-sniffs.yml

```
@@ -103,53 +26,46 @@ jobs:
     name: Run code sniffs
     runs-on: ubuntu-latest
 
+    env:
+      XMLLINT_INDENT: '	'
+
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
 
-      - name: Get Composer cache directory
-        id: composer-cache
-        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
-
-      - name: Configure Composer caching
-        uses: actions/cache@v2
-        env:
-          cache-name: cache-composer-dependencies
-        with:
-          path: ${{ steps.composer-cache.outputs.dir }}
-          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-composer-
-
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
           php-version: '7.4'
           coverage: none
 
-      - name: Install xmllint
-        run: sudo apt-get install libxml2-utils
-
-      - name: Install PHPCS
+      # Using PHPCS `master` as an early detection system for bugs upstream.
+      - name: Set PHPCS version
         run: composer require squizlabs/php_codesniffer:"dev-master" --no-update --no-scripts
 
       - name: Install Composer dependencies
-        run: composer install
+        uses: ramsey/composer-install@v1
+
+      - name: Install xmllint
+        run: sudo apt-get install libxml2-utils
+
+      # Show XML violations inline in the file diff.
+      # @link https://github.com/marketplace/actions/xmllint-problem-matcher
+      - uses: korelstar/xmllint-problem-matcher@v1
 
       # WordPress Coding Standards.
       # @link https://github.com/WordPress/WordPress-Coding-Standards
       # @link http://pear.php.net/package/PHP_CodeSniffer/
       - name: Run PHPCS ignoring warnings
-        run: vendor/bin/phpcs --runtime-set ignore_warnings_on_exit 1
+        run:  vendor/bin/phpcs --runtime-set ignore_warnings_on_exit 1
```

```suggestion
        run: vendor/bin/phpcs --runtime-set ignore_warnings_on_exit 1
```

Extra space was here on purpose or a typo?

---

Thanks for catching that. That was a typo for sure.